test/activation_based/test_flex_sn_inductor.py

```
@@ -143,6 +236,188 @@ def two_input_core(a, b, v):
+    x_compiled = x_eager.detach().clone()
+    v0_eager = torch.zeros(N)
+    v0_compiled = v0_eager.detach().clone()
+
+    def run_scan(x, v0):
+        with torch.no_grad():
+            return lowerable_while_loop_scan(_lif_core, 1, 1, 1, x, v0)
+
+    eager_out = run_scan(x_eager, v0_eager)
+    compiled_out = torch.compile(run_scan, fullgraph=True)(x_compiled, v0_compiled)
+
+    for compiled_tensor, eager_tensor in zip(compiled_out, eager_out):
```

Ruff (0.15.10)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)